### DISCUSSION TITLE
Chatting with multiple models simultaneously

### DISCUSSION THREAD

**Discussion by ross-rosario** [ggozad/oterm#109] [Ideas]
This is certainly just a nice-to-have in such a powerful llm chatting app.

- Not necessarily a MoA or anything fancy, just the ability to send a prompt to multiple models and see the responses side by side.
- Ollama supports parallel execution, so with a couple of small models, this is quite feasible on a half decent GPU.

**Comment by ggozad** [comment]
It's kind of difficult to see how this could work UI-wise...
I don't think I can prioritize this but if there is interest from someone else building it will be happy to help.

**Reply by ross-rosario** [reply]
Absolutely. Just keeping the idea out there shall other users find it useful and be able to chime in with ideas. Just for the reference, it looks like this in openwebui:
<img width="1363" alt="Screen Shot 2024-09-04 at 8 06 04 AM" src="https://github.com/user-attachments/assets/e6e9b3c2-06b9-4134-9cc4-dc8c04740bfc">
